pkg/manager/manager.go

```diff
@@ -107,6 +108,8 @@ func Login(ctx context.Context, password, address string) (user string, pass str
 			} else {
 				logrus.Debugf("tokenRequest.Status.Token is empty")
 			}
+		} else if passwordIsSpecified && strings.Contains(err.Error(), "404") {
+			return "", "", fmt.Errorf("specified token does not exist, please create one and try again")
```

Comment:
For cli users, does "create one and try again" really mean "get a new token from the manager UI"? Is there another way to do that right now?

Reply:
Yeah, that essentially means go to the UI and get one or run `acorn login <manager env>` (since there is a flow for that)

Reply:
What do you think about this?

```
specified token does not exist; please create a token via the web UI or omit the --password flag to request one via your browser
```

Reply:
lgtm!